.golangci.yml

```diff
@@ -116,25 +118,26 @@ linters-settings:
     allow-separated-leading-comment: false
 
 linters:
+  disable-all: true
   enable:
-    - bodyclose
+    # - bodyclose # too many false positives (we close the response in Parse, but generate it in Run)
     - dupl
     - funlen
     - gochecknoinits
     - gocognit
     - goconst
     - gocritic
     - gocyclo
-    - golint
+    - revive
     - gofmt
     - goimports
     - goprintffuncname
     - gosec
     #- lll
-    - maligned
+    # - fieldalignment # Disabled temporairly
```

Comment:
Same here

Reply:
This will be enabled later on, when we will require all `struct`s to be packed as fieldalignment suggests